### ISSUE TITLE
Mailing List Functionality

### ISSUE THREAD

**Issue by quilime** [OooShiny-Community/migration#3]
Interacting with Discourse as mailing list

 - https://meta.discourse.org/t/discourse-vs-email-mailing-lists/54298/4
 - https://meta.discourse.org/t/replacing-mailing-lists-email-in/13099
 - https://meta.discourse.org/t/start-a-new-topic-via-email/62977	
 - https://github.com/discourse/mail-receiver
 - https://meta.discourse.org/t/configuring-reply-via-email/42026
 - https://meta.discourse.org/t/straightforward-direct-delivery-incoming-mail/49487
 - https://meta.discourse.org/t/filtering-known-bad-sender-domains-from-your-mail-receiver/118760

**Comment by quilime** [comment]
- [ ] Test/Confirm incoming/outgoing mail
- [ ] Enable create-topic-by-email
- [ ] Setup receiving incoming mail to create new topics

**Comment by quilime** [comment]
Moved to #2

**Comment by quilime** [comment]
Re-opened as separate issue. Dependent on [#2]
